Sources/PackageGraph/PackageGraph.swift

```diff
@@ -21,11 +21,17 @@ public struct PackageGraph {
     public let packages: [ResolvedPackage]
 
     /// Returns list of all targets (reachable from root targets) in the graph.
-    public let targets: [ResolvedTarget]
+    public let targets: Set<ResolvedTarget>
```

Comment:
Should we call this reachableTargets?

Reply:
Good idea. Also renamed `products` to `reachableProducts`.